Scripts/pycairoInstall.py

```diff
@@ -0,0 +1,45 @@
+#This script install pycairo.
+import platform
+import os
+import sys
+import urllib.request
+
+#python 3.7 32-bit
+#pycairo‑1.19.1‑cp37‑cp37m‑win32.whl
```

Comment:
I appreciate the inclusion of these comments. However, could you pleas be more verbose here? What is the comment trying to tell me? I *think* it is trying to tell me what version of pycairo it will try to install first. If that's the case, please change it to "In case the python version is 3.7 and the system is 32-bit, try such and such version of cairo", or something to that effect. It would also help knowing why this is the appropriate version of cairo for that system.

Please do the same for all other comments in this file.